routes/accounts.js

```diff
@@ -17,14 +18,31 @@ router.get( '/register', ( request, response ) => {
   response.render( 'accounts/register' )
 })
 
-router.post( '/register', ( request, response , next) => {
+router.post( '/register', ( request, response, next ) => {
   const User = request.app.get( 'models' ).User
 
   const { email, password } = request.body
 
-  User.create({ email, password: encryptPassword( password ) })
-    .then( user => RegistrationEmail.send( user ))
-    .then( user => response.redirect( '/accounts/verify' ))
+  const registerUser = (email, password) => {
```

Comment:
Since this does not rely on parameters in this scope, let's move this function declaration out (possibly into the `verify_user` module?).